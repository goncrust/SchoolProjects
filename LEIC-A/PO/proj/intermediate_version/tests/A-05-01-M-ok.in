3
3
cliente1
Nome
090909
2
0
0
