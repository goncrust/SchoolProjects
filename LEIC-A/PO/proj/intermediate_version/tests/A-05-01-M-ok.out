MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: Identificador do cliente: Nome do cliente: NIF do cliente: GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: CLIENT|cliente1|Nome|90909|NORMAL|YES|0|0|0
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 